pkg/apis/etcd/v1beta2/backup_types.go

```diff
@@ -47,8 +47,11 @@ type EtcdBackup struct {
 
 // BackupSpec contains a backup specification for an etcd cluster.
 type BackupSpec struct {
-	// ClusterName is the etcd cluster name.
-	ClusterName string `json:"clusterName,omitempty"`
+	// EtcdEndpoints specifies the etcd cluster endpoints.
+	// etcd backup operator determines a endpoint that points
```

Comment:
when multiple endpoints are given, the backup operator retrieves the backup from the endpoint that has the most up-to-date state. the given endpoints must belong to the same etcd cluster.